src/br/com/harbitech/school/course/Course.java

```diff
@@ -0,0 +1,66 @@
+package br.com.harbitech.school.course;
+
+import br.com.harbitech.school.validation.ValidationUtil;
+import br.com.harbitech.school.subcategory.SubCategory;
+
+import java.time.OffsetDateTime;
+
+public class Course {
+
+    private Long id;
+    private String codeUrl;
+    private String name;
+    private OffsetDateTime completionTime;
```

Comment:
```suggestion
    private int completionTimeInHours;
```

Aqui o nome é mais para semantica